### ISSUE TITLE
New NormalPeoplePicker breaks aria-input-field-label rule 

### ISSUE THREAD

**Issue by XianghanWang** [microsoft/fluentui#14126]
### Environment Information

- **Package version(s)**: 7.121.12
- **Browser and OS versions**: chrome + Windows 10

### Describe the issue:
When upgrade FluentUI to 7.121.12, the new NormalPeoplePicker code changed. It is <div role='combobox' ... > ... <input role='textbox' ... /> </div> instead of  "<input role='combobox' ... />"

The combobox div doesn't have a arial-label attribute which breaks aria-input-field-label rule  https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name?application=axeAPI

### Please provide a reproduction of the issue in a codepen:

https://fabricweb.z5.web.core.windows.net/oufr/7.123.5/#/examples/peoplepicker

#### Actual behavior:

div **role="combobox" aria-expanded="false" aria-haspopup="dialog" class="ms-FocusZone css-216" data-focuszone-id="FocusZone1452">**<div class="ms-SelectionZone" role="presentation"><div class="ms-BasePicker-text text-424"><input autocapitalize="off" autocomplete="off" aria-autocomplete="both" spellcheck="false" **aria-label="**People Picker" class="ms-BasePicker-input input-362" aria-controls=" " role="textbox" data-lpignore="true" value="" tabindex="0"></div></div></div

#### Expected behavior:

div role="combobox" ** id='bluh' aria-label='People Picker'** aria-expanded="false" aria-haspopup="dialog" class="ms-FocusZone css-216" data-focuszone-id="FocusZone1452"><div class="ms-SelectionZone" role="presentation"><div class="ms-BasePicker-text text-424"><input autocapitalize="off" autocomplete="off" aria-autocomplete="both" spellcheck="false" **aria-labelledby="bluh"** class="ms-BasePicker-input input-362" aria-controls=" " role="textbox" data-lpignore="true" value="" tabindex="0"></div></div></div

### Documentation describing expected behavior
https://dequeuniversity.com/rules/axe/3.3/aria-input-field-name?application=axeAPI

**Comment by paulgildea** [comment]
@smhigley FYI

@XianghanWang What version did you upgrade from? Did you upgrade from 6 to 7 or was this a minor version update?

@joschect Based on the version response.... was there any breaking changes to the PeoplePicker?

**Comment by XianghanWang** [comment]
@paulgildea  I upgraded from 7.48.0. It is a minor version update, right?

**Comment by xugao** [comment]
Note - this bug is introduced in PR: #10872

**Comment by paulgildea** [comment]
Thanks for tracking this down @xugao !

@joschect is this something we can roll back or patch fix in the next version going out?

**Comment by JustSlone** [comment]
Possible fix being worked on in PR #16187
